monai/bundle/scripts.py

```diff
@@ -1173,6 +1175,8 @@ def _export(
     Args:
         converter: a callable object that takes a torch.nn.module and kwargs as input and
             converts the module to another type.
+        saver: a callable object that takes the converted model and a filepath as input and
```

Comment:
Here you could state the callable accepts the network to save, the path to save to, and a dictionary of extra JSON files (name -> contents) to include with this.

Reply:
Alright, your suggestion is very good. After confirming all the changes, I will clearly explain the `saver` parameters. Thank you!